nbd/src/util.cpp

```diff
@@ -441,6 +393,74 @@ int load_module(NBDConfig *cfg) {
     return ret;
 }
 
+int map_on_unused_nbd_device(int sockfd, std::string* devpath, int* nbdindex) {
```

Comment:
This function can be protect function in the parent class ’NBDController' ?

Reply:
Done